frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/FetchAllRemotesAction.java

```diff
@@ -19,6 +19,14 @@
 @CustomLog
 public class FetchAllRemotesAction extends BaseProjectDependentAction {
 
+  private static final long FETCH_ALL_UP_TO_DATE_TIMEOUT = 60 * 1000;
+
+  private static long lastFetchTimeMillis = 0;
+
+  public static boolean isUpToDate() {
+    return System.currentTimeMillis() > lastFetchTimeMillis + FETCH_ALL_UP_TO_DATE_TIMEOUT;
```

Comment:
Hmmm isn't the method name INcompatible with the condition? Since if `lastFetchTimeMillis` is, say, `0`, than the condition is satisfied but well, the action is NOT up to date :/